pkg/webhook/preflight/nutanix/clients.go

```diff
@@ -18,10 +18,25 @@ import (
 
 // client contains methods to interact with Nutanix Prism v3 and v4 APIs.
 type client interface {
-	GetCurrentLoggedInUser(ctx context.Context) (*prismv3.UserIntentResponse, error)
+	GetCurrentLoggedInUser(
+		ctx context.Context,
+	) (
+		*prismv3.UserIntentResponse,
+		error,
+	)
+
+	GetImageById(
+		ctx context.Context,
+		uuid *string,
+		args ...map[string]interface{},
```

Comment:
Can you please explain how these `args` are used with the `ctx` change?

Reply:
Good question, to which I have no answer. I don't know their purpose :cry: . But because the actual client accepts them as arguments, I didn't want to remove them from the wrapper.